src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperOutputExtensions.cs

```diff
@@ -100,5 +106,70 @@ public static class TagHelperOutputExtensions
                 tagHelperOutput.Attributes.Remove(attribute);
             }
         }
+
+        private static void CopyHtmlAttributeMaintainOrder(
+            int allAttributeIndex,
+            TagHelperOutput tagHelperOutput,
+            TagHelperContext context)
```

Comment:
Probably worth sticking with the `TagHelperOutput`, {specific attribute info}, `TagHelperContext` order of the (other) `CopyHtmlAttribute()` method.